__tests__/plugin.test.js

```diff
@@ -20,14 +22,29 @@ jest.mock('fs-extra', () => ({
   readFileSync: () => 'cheese',
   pathExistsSync: () => false,
   copySync: () => null,
-  removeSync: () => null,
+  removeSync: jest.fn().mockReturnValue(null),
```

Comment:
Could be made more terse:

```suggestion
  removeSync: jest.fn(() => null),
```

Reply:
I think I will leave this as-is because it is more consistent with other usages in this codebase.